buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchNode.java

```diff
@@ -742,7 +742,8 @@ private void runOpenSearchBinScript(String tool, CharSequence... args) {
     }
 
     private Map<String, String> getOpenSearchEnvironment() {
-        Map<String, String> defaultEnv = new HashMap<>();
+        Map<String, String> defaultEnv = System.getenv().entrySet().stream()
```

Comment:
This seems to not go into right direction, see please reasoning here https://github.com/opensearch-project/OpenSearch/pull/13689/files#diff-0eecf581460b128c3d70a559dfdabece54a006048cc0a4e773d020ce1f04fc9bL814

Reply:
I see. Then we should keep only a `PATH` variable.

Reply:
Do you agree that we should preserve at least `PATH` here?

Reply:
> Are you agree that we should preserve at least `PATH` here?

So any environment injection (as per https://github.com/opensearch-project/OpenSearch/pull/13689/files#diff-0eecf581460b128c3d70a559dfdabece54a006048cc0a4e773d020ce1f04fc9bL814) risks to harm reproducibility, but I don't really know the background of why things were done this way, @dblock do you?

Reply:
I do not.

Reply:
The original comment though is pretty clear. It just wants a clean slate of an environment to make things consistent.

Reply:
Correct, so we should not pursue the route of injecting env variables @anti-social